### PR TITLE
Added some more apps 

### PR DESCRIPTION
**Pull request by XXDRD**
As I am a daily user of these apps, I can easily say that they comply with the rules you set for adding apps.

### CODE REVIEW

**Comment by XXDRD** [comment]
mpv-android might be minimal but supports almost every feature which is present in other media players while giving out the best video quality 

I do agree it might be a little hassle to set up for newbies, but other than that, there is no flow in the app for being a  media player.

**Comment by XXDRD** [comment]
About Aniyomi :  
It just adds anime support in Tachiyomi, but the reason I added it to this list is that anime support is phenomenal. It even supports the ability to stream/play with mpv-android media player.

**Comment by Psyhackological** [comment]
It is, but I disagree with `every feature`. For example - missing features:
- Many gestures
- Last played position

We can go deeper with:
- Playlist
- Sorting by duration
etc.

Yes I understand however it is very minimal in features and compared to VLC it is just behind.

**Comment by Psyhackological** [comment]
About Aniyomi:
Oh I see, that is great. But still separate section kinda itches me.

**Comment by XXDRD** [comment]
It supports gestures and last played position. Dig a little more deep in app settings.

It does support long press to play everything in a folder as a playlist 

I never needed to sort by duration, so I can't say about that. It might lack a small amount of features compared to VLC, but it still is a viable alternative to many other media players and for those who don't need a lot of features

**Comment by XXDRD** [comment]
> About Aniyomi: Oh I see, that is great. But still separate section kinda itches me.

Ok, I have an Idea 
You can create a section of trackers 

which includes book trackers, anime trackers, manga trackers, music trackers/scrobblers, show trackers like showly and many more

**Comment by Psyhackological** [comment]
I meant "double tap gestures" and I cannot see how can I make it save last played position. Could you help me? I am testing it on mkv file.

Yeah I see, but let me stress out that playlist != folder, however you're right.

You kinda convinced me but not quite fully.

**Comment by Psyhackological** [comment]
Yes I've had that idea too but there is a problem - I do not want to nest things. I saw that Trackers could be a single section, but Anime Tracker and Book Tracker are very different and I do not know how to make it work.

**Comment by Psyhackological** [comment]
I am a Python guy and there is a rule `Flat is better than nested.` and also it's going to be a mess with MarkDown. And I do not like idea of 5% of my list being nested while 95% is not.

**Comment by Psyhackological** [comment]
So backing to the topic:
1. Any idea what to do with it without nesting?
2. How can I enable last played position on mpv-android?
3. Which is the main repo of Mull? GitLab or GitHub?
4. Did you use both of these RSS Readers?
5. And this calculator?

**Comment by XXDRD** [comment]
You could also add [this](https://divestos.org/index.php?page=recommended_apps)  to the end

**Comment by Psyhackological** [comment]
1. So please make section 
`## Anime & Manga`
And put `Tachiyomi` and `Aniyomi` in there. Or do you have a better idea?
Also you forgot to delete `[]` at Aniyomi Google Play and F-Droid.
2. Oh... there is a setting panel beside player... cool to know. You've convinced me, mpv-android is awesome! But please delete yt-dlp support for now, okay? And you can add [official page](https://mpv.io/) to mpv.
3. So please leave only GitLab. I would rather prefer main repository than a mirror one.
4. Alright I've tested them. They are fine.
5. I love this clock! How could I not know about him.
6. Awesome calculator! Nice find.
7. You keep forgetting about app counter. For now it is 102, but you've added 7 apps, so that means changing it to 109.
8. Wow I am overwhelmed with your work and effort. You can add with at the bottom in Special thanks to section.

When everything above will be met I can merge this pull requests.

File: README.md [hunk 1]
```diff
@@ -18,18 +18,17 @@
 [More detailed explanation here.](RULES.md)
 
 [Why does it exist?](WHY.md)
-
```
Comment (XXDRD) [inline README.md]: oops

**Review by XXDRD — COMMENTED** [review]
Going to sleep might add more apps tomorrow

**Comment by Psyhackological** [comment]
> Fixed noice image link
anime and manga section
removed GitHub source for mull 
removed heart emoji from the apps I  added (owner can add them himself)

I was going to fix this noice image link, thanks.
Nice, but does link to this work? It should not be `[Anime & Manga](#anime-&-manga)`?
Thanks
Well I was thinking about changing `My Personal Favourite` to `Personal Favourite`. When you use this app on daily basis and love everything about it you can add ❤ to the app.

Goodnight! Thank you for everything!

**Comment by XXDRD** [comment]
https://github.com/XXDRD/AAA#anime--manga
Does link to desired section in the list .
Idk why #anime-&-manga doesn't work.

**Review by XXDRD — COMMENTED** [review]
About the account requirement rule
This app uses user profiles which are kinda accounts but all the accounts are created locally. 
Self-hosted!!

**Comment by Psyhackological** [comment]
It is weird that works that way, but okay.

**Comment by Psyhackological** [comment]
I am not sure about Jellyfin. It is more about forcing user to type credentials to use this app so idk. I heard about Jellyfin and I know it ticks everything in the rules but account rule.

**Comment by Psyhackological** [comment]
I counted the apps just by searching for `<img` - logo and Special thanks to img.

**Comment by XXDRD** [comment]
> I am not sure about Jellyfin. It is more about forcing user to type credentials to use this app so idk. I heard about Jellyfin and I know it ticks everything in the rules but account rule. Where and how do you self host your instance?

User can host anywhere he wants. Home PC, server, old laptop, docker containers etc...

**Review by XXDRD — COMMENTED** [review]
All done by my side 

Will add more apps later

**Comment by Psyhackological** [comment]
Well let me please check everything then I can merge. So hold your horses with more apps. Make maybe separate pull request.

**Comment by XXDRD** [comment]
> Yes, I know what self-hosted means, but it requires and account and some configuration (not that easy to use compared to VLC for example). Do not know what to do with it. Maybe a separate category for self-hosted? If someone interests with self-hosting he would have know earlier what to do and what app does he need. I need to update the rules.

Jellyfin is pretty easy to set up. 
Jellyfin and VLC are fundamentally different apps. 
I had added Jellyfind to Video player category, because playstore had it there

**Comment by Psyhackological** [comment]
Move Jellyfin to '##Self-hosted' Contents category.
Package Manager is in bad alphabetically order in 'Contents'.
Jellyfin has mpv's official link and vice versa.
mpv when installed is named mpv not mpv-android, it is the same as VLC. So please change 'mpv-android' to 'mpv'
You sorted alphabetically Video Players and that is the reason of moving VLC to the bottom? To be honest, it is better not to move apps, just place yours under another. Less confusion next time someone's revisit the list.

Do that and I'm going to merge.

**Comment by Psyhackological** [comment]
I'm going to merge this anyway, but I do not understand why PDF is the highest on the list. 
``` markdown
- [PDF Viewer](#pdf-viewer)
- [Package Manager](#package-manager)
- [Password Manager](#password-manager)
- [Pomodoro Timer](#pomodoro-timer)
```

**Pull request merged**